### DISCUSSION TITLE
[濯水]莫妮卡大姐姐的新单曲你们听了吗

### DISCUSSION THREAD

**Discussion by Dongxiang233** [share121/inter-knot#668] [其他]
莫妮卡大姐姐的新单曲MV你们看了吗?
感觉好看里有一丝难听，难听里又透露着好看。
眼睛很享受，耳朵很痛苦。
<img width="273" alt="51dc17eb00a4a637ca7483a3c784299" src="https://github.com/user-attachments/assets/c1a55e8e-5126-47f6-b150-e48e003452a7">

**Comment by Yizakl** [comment]
比利已收藏

**Reply by NIUTE-SKMD** [reply]
已经买爆了
